db/migrate/20181123111150_add_consent_to_legal_aid_applications.rb

```diff
@@ -0,0 +1,6 @@
+class AddConsentToLegalAidApplications < ActiveRecord::Migration[5.2]
+  def change
+    add_column :legal_aid_applications, :open_banking_consent, :string
```

Comment:
This should be a boolean and not a string. If you keep it as a string, the code is going to be littered with string comparisons each time code need to determine whether consent was given.

Reply:
👍